src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ImageTests/LoadFromBytes.xaml.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Uno.UI.Samples.Controls;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Imaging;
+
+namespace Uno.UI.Samples.UITests.ImageTestsControl
+{
+	[SampleControlInfo("Image", "LoadFromBytes")]
+	public sealed partial class LoadFromBytes : UserControl
+	{
+		private readonly string _imageUrl = "https://www.spriters-resource.com/resources/sheet_icons/30/32521.png";
+
+		public LoadFromBytes()
+		{
+			InitializeComponent();
+			Loaded += OnLoaded;
+		}
+
+		private async void OnLoaded(object sender, RoutedEventArgs e)
+		{
+			var client = new HttpClient(new CorsByPassHandler());
```

Comment:
You do not need to download the image from the network. You can simply get the file locally in the embedded resources, and pass it as a stream to the ImageSource.

Reply:
Also, you can show the same image as an example using normal loading mechanisms, so that the person reviewing the sample can determine that the behavior is the right one.

Reply:
> You do not need to download the image from the network. You can simply get the file locally in the embedded resources, and pass it as a stream to the ImageSource.

Is there a sample on how to do that? I don't know the exact mechanism. Do I need to copy the image to the Samples project and set the build action to Embedded Resource and load it using the GetResourceManifestStream to get the stream to the image?